src/react-apps/applications/shared/src/components/AltinnInformationPaper.tsx

```diff
@@ -0,0 +1,38 @@
+import { createMuiTheme, Paper } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import * as React from 'react';
+import altinnTheme from '../theme/altinnStudioTheme';
+
+export interface IAltinnInformationPaperCompontentProvidedProps {
+  classes: any;
+}
+
+export interface IAltinnInformationPaperComponentState {
+}
+const theme = createMuiTheme(altinnTheme);
+
+const styles = {
+  paper: {
+    background: theme.altinnPalette.primary.yellowLight,
+    boxShadow: '1px 1px 4px rgba(0, 0, 0, 0.25)',
+    borderRadius: 0,
+    fontSize: 16,
+    padding: 26,
+    fontFamily: 'Roboto, Helvetica, Arial, sans-serif',
+    marginTop: 13,
```

Comment:
I suggest you don't put hard code external styling inside component.

Reply:
margin removed